include/RAJA/internal/LegacyCompatibility.hpp

```diff
@@ -180,6 +180,22 @@ RAJA_HOST_DEVICE RAJA_INLINE constexpr Result max(Args... args)
 }
 
 
+struct miner {
```

Comment:
'miner'? Like a guy working underground with a pick and wearing a headlamp?  Can we find a better name for this; e.g., get_min, min_picker?